docs/content/dagster-plus/deployment/serverless.mdx

```diff
@@ -286,6 +286,25 @@ If your organization begins to hit the limitations of Serverless, you should tra
 
 To switch to Hybrid, navigate to **Status > Agents** in your Dagster+ account. On this page, you can disable the Serverless agent on and view instructions for enabling Hybrid.
 
+After changing the deployment type, you will need to update your code locations' images and configuration to be compatible with the particular flavor Hybrid agent that you chose.
+
+Be sure to review the following steps:
+
+- Update the code location configurations in [`dagster_cloud.yaml`](/dagster-plus/managing-deployments/dagster-cloud-yaml) to adapt it as needed for the specificities of the agent you are using if necessary. See these references for help:
+
+  - [Amazon ECS agent configuration reference](/dagster-plus/deployment/agents/amazon-ecs/configuration-reference#per-location-configuration)
+  - [Kubernetes agent configuration reference](/dagster-plus/deployment/agents/kubernetes/configuration-reference#per-location-configuration)
+  - [Docker agent configuration reference](/dagster-plus/deployment/agents/docker/configuration-reference#docker-agent-configuration-reference)
+
+- Replace your current build process and use it to publish new images for your code locations and update your deployment with the new configurations.
```

Comment:
I keep getting stuck on the wording here. We're basically asking people to go through the CI/CD steps in the Getting Started guide, right?

Reply:
I guess I was mindful that not every user use our CI/CD, so I was trying to say: whatever was your process, it needs to change

Reply:
... and also stressing that it needs to be executed at least once to replace the existing images and the code locations.

```suggestion
- Update your CI/CD process and execute it to publish new container image and configuration for each code location.
```

Reply:
is it better?

Reply:
It's good to be mindful! With that context I see where this is going.

Would it be fair to say something like:

> **Update your build process** to publish a new container image and configuration for each code location. To use Dagster's CI/CD process, refer to [Step 4 of the Dagster+ getting started guide](/dagster-plus/getting-started#).

Reply:
sold